api/resource/resource_test.go

```diff
@@ -1402,31 +1402,31 @@ spec:
 `))
 	assert.NoError(t, err)
 	r.AppendRefBy(resid.FromString("knd1.ver1.gr1/name1.ns1"))
-	assert.Equal(t, r.RNode.MustString(), `apiVersion: v1
+	assert.Equal(t, `apiVersion: v1
 kind: Deployment
 metadata:
   name: clown
   annotations:
-    internal.config.kubernetes.io/refBy: knd1.ver1.gr1/name1.ns1
+    internal.config.kubernetes.io/refBy: 'knd1.ver1.gr1/name1.ns1'
```

Comment:
Are we ok with this changing? It's an effect of using `AnnotationSetter`, which is doing this on purpose. It has this inline comment where it forces the style: `// some tools get confused about the type if annotations are not quoted`. I'm thinking this is acceptable, since `appendCsvAnnotation` is only used by build annotations AFAICT.

Reply:
Yeah, I'm fine with this, since it only affects our kustomize-internal build annotations.